mesheryctl/internal/cli/root/app/onboard.go

```diff
@@ -22,14 +22,17 @@ import (
 )
 
 var (
-	skipSave bool // skip saving a app
-	appFile  string
+	skipSave   bool   // skip saving the app
+	appFile    string // app file
+	sourceType string // app file type (manifest / compose)
 )
 
+var validSourceTypes = []string{"compose", "manifest", "helm"}
```

Comment:
@gr455 These are not the source-types, we are using, Refer the doc. // @Revolyssup 
Also, these should not be hardocded. 
You can use `/api/application/types` endpoint to get valid sourcetypes. 
In `PreRunE` step you can get all the types and use.